CMakeLists.txt

```diff
@@ -113,87 +126,91 @@ set(CMAKE_C_COMPILER $ENV{CMAKE_C_COMPILER})
 set(BUILDDIR "$ENV{BUILDDIR}")
 
 if(ARCH_TYPE_OPTION)
-  if(NOT
-     (${ARCH_TYPE_OPTION} STREQUAL "x86")
-     AND NOT (${ARCH_TYPE_OPTION} STREQUAL "armv7"))
-    message(
-      AUTHOR_WARNING
-        "HW architecture is not defined, using x86. Allowed values are x86/armv7 (case sensitive)"
-      )
-    set(ARCH_TYPE_OPTION "x86")
-  endif()
+    if(NOT
+       (${ARCH_TYPE_OPTION} STREQUAL "x86")
+       AND NOT (${ARCH_TYPE_OPTION} STREQUAL "armv7"))
+        message(
+            AUTHOR_WARNING
+                "HW architecture is not defined, using x86. Allowed values are x86/armv7 (case sensitive)"
+            
```

Comment:
@vilyukh redundant empty line.

Reply:
@BSolonenko Since it breaks the command into tokens and the line break character can be in another token.
Therefore, in some cases there is a double line break. It is not possible to fix this issue since it is necessary to completely change the text processing logic of the command. That can lead to regression.

Reply:
@BSolonenko You can correct it, should run cmake-format with follow argument: --dangle-parens 1 or add in config-file as dangle_parens: True

Reply:
@MaximRybak please make this default